optuna/integration/keras.py

```diff
@@ -57,6 +57,10 @@ def on_epoch_end(self, epoch: int, logs: Optional[Dict[str, float]] = None) -> N
         logs = logs or {}
         current_score = logs.get(self._monitor)
         if current_score is None:
+            message = "There is not an evaluation metric for pruning named {}.".format(
+                self._monitor
+            )
+            warnings.warn(message)
```

Comment:
Please import `warnings` to use it.

@hvy  I think you expect an error here instead of `warnings` in https://github.com/optuna/optuna/issues/1226#issue-615516601. I guess raising the errors may break user code if the Keras returns `None` as expected values. But, I'm not sure if it occurs.
What do you think of it?

Reply:
Thank you for your reply!
I just noticed this mistake and fixed it.

Reply:
Thanks @toshihikoyanase for pinging. I think the current design to yield a warning sound good. You're right about the compatibility issue, and I also believe we have to be more careful if we are to raise errors, i.e. make sure that we understand the behavior of the integration framework.

Reply:
@hvy Thank you for your reply. That makes sense. I'll accept `warning`.